File: JS/URI/1064.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

var positivo = 0;
var soma = 0;

for (var i = 0; i < 6; i++) {
    var numero = parseFloat(lines.shift());


    if (numero > 0) {
        positivo++;
        soma += numero;
    }


}

soma = soma / positivo;

console.log(positivo + " valores positivos");
console.log(soma.toFixed(1));

File: Python/URI/1064.py
positivo = 0
soma = 0
for i in range(6):
    numero = float(input())

    if(numero > 0):
        positivo += 1
        soma+=numero

soma  = soma/positivo

print(str(positivo)+" valores positivos")
print("{:.1f}".format(soma))

